shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/presentation/module/HomeViewModel.kt

```diff
@@ -26,17 +36,43 @@ class HomeViewModel(
 
     private var currentPage = 1
 
+
+
     fun fetchData() {
-        getSurveysUseCase(currentPage, 10, isForceLatestData = false)
+        getProfile()
             .onStart { setStateLoading() }
-            .catch { emit(emptyList()) }
+            .combine(fetchSurvey(currentPage, 10, false)) { user, surveys ->
+                Pair(user, surveys)
+            }
```

Comment:
We can get rid of the `Pair` since it's not consumed.
```suggestion
            .combine(fetchSurvey(currentPage, 10, false)) { _, _ -> }
```

Reply:
solved in [32858cd](https://github.com/nimblehq/avishek-phong-kmm-ic/pull/101/commits/32858cd842897530a0caaa559295e0ad768e444a)